OWNERS_ALIASES

```diff
@@ -128,3 +128,11 @@ aliases:
   wg-resource-management-leads:
     - vishh
     - derekwaynecarr
+  steering-committee:
```

Comment:
This only includes the bootstrap members, and not the 6 elected members:
https://github.com/kubernetes/steering#members